### PR TITLE
CHE-678: improve loading factory view

### PR DESCRIPTION
**Pull request by akurinnoy**

### CODE REVIEW

**Comment by olexii4** [comment]
+1

**Comment by akurinnoy** [comment]
+1

**Comment by benoitf** [comment]
+1

File: src/app/factories/load-factory/load-factory.html [hunk 1]
```diff
@@ -18,10 +18,57 @@
...
+                </div>
+              </che-accordion>
+            </div>
+          </div>
+        </div>
+        <!--bottom bar-->
+        <div class="che-loader-bottom-bar">
+          <div layout="row" layout-align="center stretch" class="main-page">
+            <div flex="50" layout="column" layout-align="start start">
+              <che-link ng-click="backToDashboard()" che-link-text="&larr; Back to Dashboard"
```
Comment (ashumilova) [inline src/app/factories/load-factory/load-factory.html]: backToDashboard() - where is this method defined?

File: src/app/factories/load-factory/load-factory.html [hunk 2]
```diff
@@ -18,10 +18,57 @@
...
+                        class="load-factory-back-link"></che-link>
+            </div>
+            <div flex="50">
+              <div class="load-factory-retry-block"
+                   layout="column" layout-align="start end"
+                   ng-show="loadFactoryCtrl.getLoadingSteps()[loadFactoryCtrl.getCurrentProgressStep()].hasError">
+                <che-button-danger che-button-title="Retry"
+                                   ng-click="loadFactoryCtrl.resetLoadFactoryInProgress()"></che-button-danger>
+                <che-link class="load-factory-download-link"
+                          ng-click="downloadLogs(loadFactoryCtrl.getLoadingSteps()[loadFactoryCtrl.getCurrentProgressStep()].logs)"
```
Comment (ashumilova) [inline src/app/factories/load-factory/load-factory.html]: downloadLogs - where is  this method defined?

**Comment by ashumilova** [comment]
Ok

**Pull request merged**